deploy/cr.yaml

```diff
@@ -145,7 +145,39 @@ spec:
   port: "5432"
 # The number of replicas to create after a PostgreSQL primary
 # is first initialized. This only works on create
-  replicas: "0"
+#  replicas: "0"
+  pgReplicas:
+    hotStandby:
+      size: 0
```

Comment:
we have several issues here:
1) when we scale down the replicas from 3 to 1 old pvc are present
2) I have tried to scale up to 3 then scale down to 1 and after that again up to 3 but I have only 2 replica nodes :(

Reply:
https://github.com/percona/percona-postgresql-operator/pull/50/commits/35d324887677775fcc5696fb0d5c41290cdd92d0